gym_snake/envs/snakeGameGym.py

```diff
@@ -16,19 +18,37 @@ class SnakeGameGym(SnakeGame):
 	Inherits the SankeGame class that runs the Snake Game.
 	"""
 
-	def __init__(self, fps: int):
+	def __init__(self, fps: int, use_pygame: bool = True):
 		"""
 		Initializes the SnakeGameGATest class.
 		"""
+		self.use_pygame = use_pygame
 		self.move_map = {
 			0: "left",
 			1: "up",
 			2: "right",
 			3: "down",
 		}
+		
+		self.width = 500
```

Comment:
Is there a particular reason for adding all of these instance variables to the SnakeGameGym class proper? I'm fine with it, but it seems to be adding duplicate code that isn't super necessary.